src/chocolatey.tests/infrastructure.app/commands/ChocolateyHelpCommandSpecs.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright © 2017 - 2018 Chocolatey Software, Inc
+// Copyright © 2011 - 2017 RealDimensions Software, LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// 
+// You may obtain a copy of the License at
+// 
+// 	http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace chocolatey.tests.infrastructure.app.commands
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using chocolatey.infrastructure.app.attributes;
+    using chocolatey.infrastructure.app.commands;
+    using chocolatey.infrastructure.app.configuration;
+    using chocolatey.infrastructure.app.services;
+    using chocolatey.infrastructure.commandline;
+    using Moq;
+    using Should;
+
+    public class ChocolateyHelpCommandSpecs
+    {
+        public abstract class ChocolateyHelpCommandSpecsBase : TinySpec
+        {
+            protected ChocolateyHelpCommand command;
+            protected Mock<IChocolateyPackageService> packageService = new Mock<IChocolateyPackageService>();
+            protected ChocolateyConfiguration configuration = new ChocolateyConfiguration();
+
+            public override void Context()
+            {
+                configuration.Sources = "bob";
+                command = new ChocolateyHelpCommand(packageService.Object);
+            }
+        }
+
+        public class when_implementing_command_for : ChocolateyHelpCommandSpecsBase
+        {
+            private List<string> results;
+
+            public override void Because()
+            {
+                results = command.GetType().GetCustomAttributes(typeof(CommandForAttribute), false).Cast<CommandForAttribute>().Select(a => a.CommandName).ToList();
+            }
+
+            [Fact]
+            public void should_implement_help()
+            {
+                results.ShouldContain("help");
+            }
+
+        }
+
+        public class when_configurating_the_argument_parser : ChocolateyHelpCommandSpecsBase
+        {
+            private OptionSet optionSet;
+
+            public override void Context()
+            {
+                base.Context();
+                optionSet = new OptionSet();
+            }
+
+            public override void Because()
+            {
+                command.configure_argument_parser(optionSet, configuration);
+            }
+
+        }

```

Comment:
I see there are no facts being tested here - was that the same in the other class? You can likely remove this since there isn't much to configure.